packages/contracts/contracts/interfaces/IChugSplashManager.sol

```diff
@@ -1,20 +1,19 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.9;
 
+import { ChugSplashRegistry } from "../ChugSplashRegistry.sol";
 import { Version } from "../Semver.sol";
 
 /**
  * @title ChugSplashManager
  * @notice Interface that must be inherited the ChugSplash manager.
  */
 interface IChugSplashManager {
-    function activeBundleId() external returns (bytes32);
-
-    function proposers(address) external returns (bool);
-
```

Comment:
I removed these two fields because this interface should remain consistent for all future versions of the ChugSplashManager, and I wasn't 100% sure we'd use these in the future